sysutils/pfSense-pkg-nut/files/etc/inc/priv/nut.priv.inc

```diff
@@ -15,7 +17,7 @@
 	   notice, this list of conditions and the following disclaimer in the
 	   documentation and/or other materials provided with the distribution.
 
-	THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+	THIS SOFTWARE IS PROVIDED "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
```

Comment:
It wasn't a change per se, it's actually a completely new file with the same name. All the files in the new package have the same copyright using double quotes.

Why is there a preference for  ticks vs double quotes?


> On Jul 26, 2016, at 05:51, Renato Botelho <notifications@github.com> wrote:
> 
> In sysutils/pfSense-pkg-nut/files/etc/inc/priv/nut.priv.inc:
> 
> > @@ -15,7 +17,7 @@
> >  	   notice, this list of conditions and the following disclaimer in the
> >  	   documentation and/or other materials provided with the distribution.
> >  
> > -	THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
> > +	THIS SOFTWARE IS PROVIDED "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
> Please do not change license text. I'll import it to unbreak the ports build and will revert it after. Thanks
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>